crates/extension/src/wasm_host/wit/since_v0_0_6.rs

```diff
@@ -325,35 +324,18 @@ impl ExtensionImports for WasmState {
                         .await?;
                 }
                 DownloadedFileType::GzipTar => {
-                    let body = GzipDecoder::new(body);
                     futures::pin_mut!(body);
                     self.host
                         .fs
-                        .extract_tar_file(&destination_path, Archive::new(body))
+                        .extract_tar_gz_file(&destination_path, body)
                         .await?;
                 }
                 DownloadedFileType::Zip => {
-                    let file_name = destination_path
-                        .file_name()
-                        .ok_or_else(|| anyhow!("invalid download path"))?
-                        .to_string_lossy();
-                    let zip_filename = format!("{file_name}.zip");
-                    let mut zip_path = destination_path.clone();
-                    zip_path.set_file_name(zip_filename);
-
                     futures::pin_mut!(body);
-                    self.host.fs.create_file_with(&zip_path, body).await?;
-
-                    let unzip_status = std::process::Command::new("unzip")
```

Comment:
@maxdeviant 

How about the part? 

I make `extract_zip` method before , because here it requires the new zip extraction method, the `unzip` command is not working on Windows. And node_runtime and more places also need to.